src/libs/ajax.js

```diff
@@ -668,9 +668,14 @@ export const Files = {
 };
 
 export const Summary = {
-  getFile: async (URI, nameFile) => {
+  getFile: async (fileName) => {
+    const URI = `/consent/cases/summary/file?fileType=${fileName}`;
     const url = `${await Config.getApiUrl()}${URI}`;
-    return await getFile(url, nameFile);
+    if (fileName === 'TranslateDUL') {
```

Comment:
Instead of using an if-else block, you can create an object that maps the keys to the filenames as a const reference (something like `{'TranslateDUL': 'DUL_summary.tsv'...}`). This way you can get the filename via key/parameter as well as have an easy way to add new key-filename pairings in the future.